tests/database_tests/test_insert.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+from src import db
+
+from src.database.models import Alerts, AnomalyEquations, EmailInformation, DeviceInformation
+
+
+class TestInsert(unittest.TestCase):
+    """Tests insertion of objects into database tables using sqlalchemy"""
+
+    def setUp(self) -> None:
+        self.db = db
+        self.db.create_session()
+        self.alert = Alerts(alert_type='IDS', description='test', severity=1, mac_address='00:00:00:00:00',
+                            payload='foobar')
+        self.anamoly_eq = AnomalyEquations(average_equation="test", adjustment_equation="test")
+        self.email_info = EmailInformation(recipient_addresses='test@email.com, foo@bar.com',
+                                           sender_address='openwrt@alert.com',
+                                           sender_email_password='super_secure_password',
+                                           smtp_server='smtp.test.com')
+        self.device_info = DeviceInformation(mac_address='00:00:00:00:00:00',
+                                             device_name='test',
```

Comment:
Ran into an issue with tests not passing, column names don't match the object's values (device_name != name. etc.)